DisguiseUnityRenderStream/Runtime/Disguise.RenderStream.asmdef

```diff
@@ -24,12 +24,12 @@
         {
             "name": "com.unity.render-pipelines.high-definition",
             "expression": "13.1.8",
-            "define": "HDRP_13_1_8_OR_NEWER"
+            "define": "HDRP"
```

Comment:
We're still checking for package version >= 13.1.8 here but this is not apparent in the define. Is this preferable?

Reply:
Having the version implicit in the asmdef like this was suggested by Etienne. Some packages include the versions in the define for compatibility logic when they support ex multiple versions of HDRP.

Reply:
OK, I'll defer to @etienne-p on this one.